UPGRADE-15.0.md

```diff
@@ -1318,3 +1318,8 @@ On Chrome there was issue when clicking on next/previous button while slider is
 #### change free transport limit in demo data ([#3199](https://github.com/shopsys/shopsys/pull/3199))
 
 -   social links in footer were blacked-out for cypress snapshots as they caused issues, so do not forget to add the blackout to snapshots where these links are visible, and also regenerate your screenshots
+
+#### add more OG meta tags ([#3228](https://github.com/shopsys/shopsys/pull/3228))
```

Comment:
I suggest avoiding the abbreviations... open graph will be more understandable

Reply:
Ok makes sense, adjusted.